src/domains/ecommerce-common/03_storage.tf

```diff
@@ -20,7 +20,7 @@ module "ecommerce_storage_snet" {
   ]
 }
 
-resource "azurerm_private_endpoint" "storage_private_endpoint" {
+resource "azurerm_private_endpoint" "storage_transient_private_endpoint" {
   count = var.env_short != "d" ? 1 : 0
 
   name                = "${local.project}-storage-private-endpoint"
```

Comment:
```suggestion
  name                = "${local.project}-transient-storage-private-endpoint"
```

Reply:
Your suggested rename make sense, but the name of the resource is greater than 24 chars, so i did this mapping:
tr for transient -> ${local.project}-tr-storage-private-endpoint 
dl for deadletter -> ${local.project}-dl-storage-private-endpoint 
Remember also that this strings have the - char removed